vignettes/userguide.Rmd

```diff
@@ -565,6 +568,13 @@ In this file, `#PBS-*` defines command-line arguments to the `qsub` program.
   in curly braces (`{{ ... }}`), as they are used to fill in the right
   variables.
 
+Or simply set `clustermq.scheduler` to use a default PBS template we have 
```

Comment:
I will rewrite this in the UG>templates so that it's clearer we use the default template as is quoted, and it can be changed using `options(...)` for all schedulers (not just PBS/Torque).

Reply:
In that case, I won't include it